docs/users-guide/14-x-rays.md

```diff
@@ -24,13 +24,13 @@ To quickly make your new dashboard visible to other users, go to the collection
 
 One great way to explore your data in general in Metabase is to click on points of interest in charts or tables, which shows you ways to further explore that point. We've added x-rays to this action menu, so if you for example find a point on your line chart that seems extra interesting, give it a click and x-ray it! We think you'll like what you see.
 
-![X-ray action in drill-through menu](.images/x-rays/drill-through.png)
+![X-ray action in drill-through menu](./images/x-rays/drill-through.png)
 
 ### X-rays in the Data Reference
 
 You can also create an x-ray by navigating to a table, field, metric, or segment in the [Data Reference](./12-data-model-reference.md). Just click the x-ray link in the left sidebar.
 
-![Data Reference x-ray](.images/x-rays/data-reference.png)
+![Data Reference x-ray](./images/x-rays/data-reference.png)
```

Comment:
@salsakran these popped up when I switched the targ from 29.0 to 29.1

Are these changes ok?

Reply:
Hmmm. I branched 0.29.1 off 0.29.0 and those went in when we found broken links during the release checks. 

The ones with the slash are definitely correct.